bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -105,34 +104,29 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 <!-- Ocean grids -->
 <scripgriddata hgrid="0.33x0.33"  lmask="0.33x0.33" >lnd/clm2/mappingdata/grids/SCRIPgrid_0.33x0.33_navy_c111207.nc</scripgriddata>
-<scripgriddata hgrid="5x5_amazon" lmask="navy"      >lnd/clm2/mappingdata/grids/SCRIPgrid_5x5_amazon_navy_c111207.nc</scripgriddata>
-<scripgriddata hgrid="gx1v6"      lmask="gx1v6"     >/glade/proj3/cseg/mapping/grids/gx1v6_090205.nc</scripgriddata>
-<scripgriddata hgrid="gx3v7"      lmask="gx3v7"     >/glade/proj3/cseg/mapping/grids/gx3v7_090903.nc</scripgriddata>
-<scripgriddata hgrid="tx1v1"      lmask="tx1v1"     >/glade/proj3/cseg/mapping/grids/tx1v1_090122.nc</scripgriddata>
-<scripgriddata hgrid="tx0.1v2"    lmask="tx0.1v2"   >/glade/proj3/cseg/mapping/grids/tx0.1v2_090127.nc</scripgriddata>
+<scripgriddata hgrid="5x5_amazon" lmask="navy"      >share/meshes/5x5pt-amazon_navy_ESMFmesh_cd5_c20210107.nc</scripgriddata>
+<scripgriddata hgrid="gx1v6"      lmask="gx1v6"     >share/meshes/gx1v6_090205_ESMFmesh.nc</scripgriddata>
+<scripgriddata hgrid="gx3v7"      lmask="gx3v7"     >share/meshes/gx3v7_120309_ESMFmesh.nc</scripgriddata>
+<scripgriddata hgrid="tx1v1"      lmask="tx1v1"     >share/meshes/tx1v1_ESMFMesh_cd5_c20210105.nc</scripgriddata>
+<scripgriddata hgrid="tx0.1v2"    lmask="tx0.1v2"   >share/meshes/tx0.1v2_ESMFmesh_cd5_c20210105.nc</scripgriddata>
```

Comment:
Everything that says `scripgriddata` (above) I suggest changing to `dstmeshdata` because it corresponds to DST mesh files (ultimately fsurdat).

Everything that says `unstructdata` (below) I suggest changing to `srcmeshdata` because it corresponds to SRC mesh files (ultimately the raw datasets).

Reply:
Hmmm, the one thing I wonder there is that whether a mesh is a source or destination depends on context. There is for example a f09 grid that used as a source grid, for raw data in mksurfdata, but of course there's also the model destination grid. But, whether it's a src or dst grid depends on context, not on anything intrinsic in the grid itself.

So I would suggest just using meshdata.

Reply:
> [...] but I think it will be clearest if we go with @slevisconsulting 's proposal of listing them separately and explicitly for each. It seems okay to me if the same mesh file is listed in two places – both as a src mesh and a dst mesh – and my gut feeling is that long-term maintenance will be easier that way: e.g., if we tweak the mesh for the f09 model resolution, we would want to change the dst mesh, but NOT the src meshes. Perhaps most importantly, I think @slevisconsulting 's proposal gives a smoother transition to the rework that @mvertens is doing (which, @ekluzek , you probably haven't seen yet, so I know it's hard for you to comment on this with that goal in mind).

@ekluzek @billsacks 
Bill's comment is correct: In this PR I'm already pointing to different SRC vs. DST mesh files for the same resolution. To explain:
- The UNSTRUCT SRC mesh files -- I generated those from our SCRIP SRC files.
- The DST mesh files -- I'm pointing to existing files that @mvertens suggested I use from here:
https://github.com/ESMCI/ccs_config_cesm/blob/main/component_grids_nuopc.xml